search/django/weedid/views.py

```diff
@@ -16,3 +30,139 @@ def elasticsearch_query(request):
         headers=request.headers,
     )
     return HttpResponse(elasticsearch_response)
+
+
+@csrf_exempt
+def upload(request):
```

Comment:
We need to make sure that all the upload views (except for list) are for logged in users only.